helpers/mail/mail_v3.go

```diff
@@ -725,5 +737,17 @@ func ParseEmail(emailInfo string) (*Email, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	if len(e.Address) > maxEmailLength {
+		return nil, errors.New("invalid email length")
+	}
+
+	parts := strings.Split(e.Address, "@")
+	local, domain := parts[0], parts[1]
+
+	if len(domain) > maxEmailDomainLength || len(local) > maxEmailLocalLength {
+		return nil, errors.New("invalid email length")
```

Comment:
I think we should specify a bit more detail about the violation. Something like: "Invalid email length, you have used X out of Y characters <link to RFC>)

Reply:
I agree. Error message should have more information.

Reply:
@thinkingserious I have improved the error messages :)